src-tauri/src/main.rs

```diff
@@ -67,9 +75,12 @@ fn build_menu() -> Menu {
         Menu::new().add_item(CustomMenuItem::new("view_toggle_preview", "Toggle Preview")),
     );
 
-    Menu::new()
-        .add_submenu(application_menu)
-        .add_submenu(file_submenu)
+    let mut menu = Menu::new();
+    #[cfg(target_os = "macos")]
+    {
+        menu = menu.add_submenu(application_menu)
+    }
+    menu.add_submenu(file_submenu)
```

Comment:
Correct me if I'm wrong, but the `Quit` action still shows up in the File submenu on macOS, and it is also still using the `file_quit` custom action on other platforms.